### PR TITLE
Anim bugfixes

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
fix: The shake now occurs at the same time as the movement, and it now doesn't trigger on containers or uplinks.
/:cl:

### CODE REVIEW

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -295,7 +295,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	pickup(user)
 	add_fingerprint(user)
-	if(!user.put_in_active_hand(src))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items.dm]: So um... does this make it so nothing does the animation?

**Comment by ExcessiveUseOfCobblestone** [comment]
where is the shake removal :^)?

**Comment by Iamgoofball** [comment]
@optimumtact can you lock discussion on this PR, it doesn't need comments from irrelevant users

**Comment by optimumtact** [comment]
@Iamgoofball conflicts

**Comment by Iamgoofball** [comment]
h o w

**Pull request merged**